pkg/component/data/instillartifact/v0/config/tasks.json

```
@@ -184,7 +184,7 @@
                   "$ref": "#/$defs/file"
                 },
                 "file-name": {
-                  "description": "The name of the file, please remember to add the file extension in the end of file name. e.g. 'example.pdf'",
+                  "description": "The name of the file, please remember to add the file extension in the end of file name. e.g. 'example.pdf'. The length limitation of file name is 100 characters.",
```

nit: 
- There are slight differences in style for the different field descriptions. In this case, this is the only description that stars by "The ..." which produces a small style inconsistency. 
- Comma isn't the best separator between the definition of the field the part about the extension.

```suggestion
                  "description": "Name of the file, including the extension (e.g. `example.pdf`). The length of this field is limited to 100 characters.",
```